src/video-player.utils.js

```diff
@@ -127,4 +126,10 @@ export const overrideDefaultVideojsComponents = () => {
     // Position the 'logo-button' button last
     children.push('logoButton');
   }
+
+  const SeekBar = videojs.getComponent('SeekBar');
+  if (SeekBar && !SeekBar.prototype.options_.children.includes('mouseTimeDisplay')) {
+    SeekBar.prototype.options_.children.splice(1, 0, 'mouseTimeDisplay');
+  }
```

Comment:
what is this doing? a comment would really help

Reply:
videojs team made a decision to remove this component on mobile devices. we want it there since we're attaching the chapter display to it. I'll add a comment.